deckhouse-controller/pkg/controller/deckhouse-release/check_release.go

```diff
@@ -545,9 +559,9 @@ func (dcr *DeckhouseReleaseChecker) nextVersion(actual, target *semver.Version)
 		return target, nil
```

Comment:
And here we just took the next minor version and return it without checking whether it is in the repository or not. I think it is worth removing this check, since below there is a check by tags that we take from the repository and there we work with the current information

Reply:
We couldn't get rid of this check completely, because we have a case where we have to return the next version within one minor version, but now the result always depends on the tags available in the registry